src/Surfnet/StepupRa/RaBundle/Form/Type/SearchRaCandidatesType.php

```diff
@@ -55,6 +61,7 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
             'data_class' => 'Surfnet\StepupRa\RaBundle\Command\SearchRaCandidatesCommand',
+            'is_sraa' => false,
```

Comment:
What use does the is_sraa default value serve?

Reply:
That should have been removed, was used as debug code. Removed